tox.ini

```diff
@@ -24,6 +24,7 @@ commands = {posargs:py.test -s -x -f -v}
 description = invoke sphinx-build and try to build the HTML page, and check that URIs are valid
 basepython = python
 deps = sphinx >= 1.6.3, < 2
+       towncrier >= 17.8.0, < 18
```

Comment:
towncrier uses calver, so that version block is not sensible

Reply:
true, will remove it!